src/puptoo/app.py

```diff
@@ -71,6 +72,19 @@ def handle_signal(signal, frame):
     global running
     running = False
 
+def redis_client() :
```

Comment:
Use two blank lines between function definitions and surrounding code.

Reply:
Done!